ui/src/app/components/gke-app-configuration-panels/create-gke-app.tsx

```diff
@@ -215,6 +220,23 @@ export const CreateGkeApp = ({
   const showDeleteDiskButton = unattachedDiskExists(app, disk);
   const showDeleteAppButton = canDeleteApp(app);
 
+  const canModifyDiskSize = !!app || unattachedDiskExists(app, disk);
+
+  const disableDiskSizeContent = cond(
+    [
+      !!app,
+      `${toUIAppType[appType]} environment already exists. To update disk size please delete disk and recreate the 
```

Comment:
I like this but I don't know that we need to be specific about which app it is here, because we're already in the panel for that app.

I'd also like an explicit declaration like "cannot do X when Y" to start this text.

Reply:
I would prefer to keep the environment name to avoid any confusion with machine configuration. Do be explicit that this environment is the problem.

 i have removed the App name for the other case i.e no app but disk exist